source/common/http/conn_manager_impl.cc

```diff
@@ -1043,6 +1044,12 @@ void ConnectionManagerImpl::ActiveStream::encode100ContinueHeaders(
     }
   }
 
+  if (!response_metadata_map_->empty()) {
```

Comment:
Looks like we're doing this more than once.   Utility function?

Reply:
Fixed. Thanks!